.github/workflows/release.yml

```
@@ -0,0 +1,94 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
```

```suggestion
        uses: actions/setup-node@v3
```
v3 also includes caching support https://github.com/actions/setup-node#caching-global-packages-data